--- UFconfig/UFconfig.mk.orig	2011-12-08 20:19:48.000000000 +0100
+++ UFconfig/UFconfig.mk	2012-01-09 16:20:56.743005449 +0100
@@ -53,7 +53,7 @@
 # default make environment.
 
 # C and C++ compiler flags.  The first three are standard for *.c and *.cpp
-CF = $(CFLAGS) $(CPPFLAGS) $(TARGET_ARCH) -O3 -fexceptions -fPIC
+CF = $(CFLAGS) $(CPPFLAGS) $(TARGET_ARCH) -fPIC
 
 # ranlib, and ar, for generating libraries.  If you don't need ranlib,
 # just change it to RANLAB = echo
@@ -67,15 +67,16 @@
 
 # Fortran compiler (not required for 'make' or 'make library')
 F77 = gfortran
-F77FLAGS = $(FFLAGS) -O
+F77FLAGS = $(FFLAGS)
 F77LIB =
 
 # C and Fortran libraries
 LIB = -lm
 
 # For "make install"
-INSTALL_LIB = /usr/local/lib
-INSTALL_INCLUDE = /usr/local/include
+INSTALL_PREFIX = /usr
+INSTALL_LIB = $(INSTALL_PREFIX)/lib
+INSTALL_INCLUDE = $(INSTALL_PREFIX)/include
 
 # For compiling MATLAB mexFunctions (MATLAB 7.5 or later)
 MEX = mex -O -largeArrayDims -lmwlapack -lmwblas
@@ -148,8 +149,8 @@
 # The path is relative to where it is used, in CHOLMOD/Lib, CHOLMOD/MATLAB, etc.
 # You may wish to use an absolute path.  METIS is optional.  Compile
 # CHOLMOD with -DNPARTITION if you do not wish to use METIS.
-METIS_PATH = ../../metis-4.0
-METIS = ../../metis-4.0/libmetis.a
+METIS_PATH = /usr/include/metis
+METIS = -lmetis
 
 #------------------------------------------------------------------------------
 # UMFPACK configuration:
