--- SuiteSparse_config/SuiteSparse_config.h.orig	2014-07-16 23:35:20.000000000 +0200
+++ SuiteSparse_config/SuiteSparse_config.h	2014-11-21 17:12:47.821093628 +0100
@@ -38,14 +38,14 @@
 #ifndef SUITESPARSE_CONFIG_H
 #define SUITESPARSE_CONFIG_H
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
 #include <limits.h>
 #include <stdlib.h>
 #include <math.h>
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 /* ========================================================================== */
 /* === SuiteSparse_long ===================================================== */
 /* ========================================================================== */
