diff -burNp UFconfig/UFconfig.mk UFconfig-dud/UFconfig.mk
--- UFconfig/UFconfig.mk	2008-09-22 20:19:37.000000000 +0200
+++ UFconfig-dud/UFconfig.mk	2009-01-23 12:25:23.482152935 +0100
@@ -122,8 +122,8 @@ XERBLA = 
 # The path is relative to where it is used, in CHOLMOD/Lib, CHOLMOD/MATLAB, etc.
 # You may wish to use an absolute path.  METIS is optional.  Compile
 # CHOLMOD with -DNPARTITION if you do not wish to use METIS.
-METIS_PATH = ../../metis-4.0
-METIS = ../../metis-4.0/libmetis.a
+METIS_PATH = /usr/include/metis
+METIS = -lmetis
 
 # If you use CHOLMOD_CONFIG = -DNPARTITION then you must use the following
 # options:
