--- UFconfig/Makefile.orig	2011-12-14 19:09:00.000000000 +0100
+++ UFconfig/Makefile	2012-01-09 16:45:18.549721061 +0100
@@ -8,15 +8,13 @@
 
 include UFconfig.mk
 
-ccode: libufconfig.a
+ccode: libufconfig.la
 
-all: libufconfig.a
+all: libufconfig.la
 
-libufconfig.a: UFconfig.c UFconfig.h
-	$(CC) $(CF) -c UFconfig.c
-	$(ARCHIVE) libufconfig.a UFconfig.o
-	$(RANLIB) libufconfig.a
-	- $(RM) UFconfig.o
+libufconfig.la: UFconfig.c UFconfig.h
+	libtool --mode=compile --tag=CC $(CC) $(CF) -c UFconfig.c
+	libtool --mode=link --tag=CC $(CC) $(LDFLAGS) $(CFLAGS) -o libufconfig.la UFconfig.lo -rpath $(INSTALL_LIB)
 
 distclean: purge
 
@@ -28,14 +26,12 @@
 
 # install UFconfig
 install:
-	$(CP) libufconfig.a $(INSTALL_LIB)/libufconfig.$(VERSION).a
-	( cd $(INSTALL_LIB) ; ln -sf libufconfig.$(VERSION).a libufconfig.a )
-	$(CP) UFconfig.h $(INSTALL_INCLUDE)
-	chmod 644 $(INSTALL_LIB)/libufconfig*.a
-	chmod 644 $(INSTALL_INCLUDE)/UFconfig.h
+	libtool --mode=install $(CP) libufconfig.la $(DESTDIR)$(INSTALL_LIB)
+	$(CP) UFconfig.h $(DESTDIR)$(INSTALL_INCLUDE)
+	chmod 644 $(DESTDIR)$(INSTALL_INCLUDE)/UFconfig.h
 
 # uninstall UFconfig
 uninstall:
-	$(RM) $(INSTALL_LIB)/libufconfig*.a
-	$(RM) $(INSTALL_INCLUDE)/UFconfig.h
+	$(RM) $(DESTDIR)$(INSTALL_LIB)/libufconfig.la $(DESTDIR)$(INSTALL_LIB)/libufconfig.so* $(DESTDIR)$(INSTALL_LIB)/libufconfig.a
+	$(RM) $(DESTDIR)$(INSTALL_INCLUDE)/UFconfig.h
 
