--- SuiteSparse_config/Makefile.orig	2012-05-27 00:53:19.000000000 +0200
+++ SuiteSparse_config/Makefile	2012-06-26 19:54:44.061969831 +0200
@@ -8,17 +8,15 @@
 
 include SuiteSparse_config.mk
 
-ccode: libsuitesparseconfig.a
+ccode: libsuitesparseconfig.la
 
-all: libsuitesparseconfig.a
+all: libsuitesparseconfig.la
 
-library: libsuitesparseconfig.a
+library: libsuitesparseconfig.la
 
-libsuitesparseconfig.a: SuiteSparse_config.c SuiteSparse_config.h
-	$(CC) $(CF) -c SuiteSparse_config.c
-	$(ARCHIVE) libsuitesparseconfig.a SuiteSparse_config.o
-	$(RANLIB) libsuitesparseconfig.a
-	- $(RM) SuiteSparse_config.o
+libsuitesparseconfig.la: SuiteSparse_config.c SuiteSparse_config.h
+	libtool --mode=compile --tag=CC $(CC) $(CF) -c SuiteSparse_config.c
+	libtool --mode=link --tag=CC $(CC) $(LDFLAGS) $(CFLAGS) -o libsuitesparseconfig.la SuiteSparse_config.lo -lrt -rpath $(INSTALL_LIB)
 
 distclean: purge
 
@@ -30,14 +28,12 @@
 
 # install SuiteSparse_config
 install:
-	$(CP) libsuitesparseconfig.a $(INSTALL_LIB)/libsuitesparseconfig.$(VERSION).a
-	( cd $(INSTALL_LIB) ; ln -sf libsuitesparseconfig.$(VERSION).a libsuitesparseconfig.a )
-	$(CP) SuiteSparse_config.h $(INSTALL_INCLUDE)
-	chmod 644 $(INSTALL_LIB)/libsuitesparseconfig*.a
-	chmod 644 $(INSTALL_INCLUDE)/SuiteSparse_config.h
+	libtool --mode=install $(CP) libsuitesparseconfig.la $(DESTDIR)$(INSTALL_LIB)
+	$(CP) SuiteSparse_config.h $(DESTDIR)$(INSTALL_INCLUDE)
+	chmod 644 $(DESTDIR)$(INSTALL_INCLUDE)/SuiteSparse_config.h
 
 # uninstall SuiteSparse_config
 uninstall:
-	$(RM) $(INSTALL_LIB)/libsuitesparseconfig*.a
-	$(RM) $(INSTALL_INCLUDE)/SuiteSparse_config.h
+	$(RM) $(DESTDIR)$(INSTALL_LIB)/libsuitesparseconfig.la
+	$(RM) $(DESTDIR)$(INSTALL_INCLUDE)/SuiteSparse_config.h
 
