--- UFconfig/UFconfig.mk.orig	2007-10-24 15:55:44.000000000 +0200
+++ UFconfig/UFconfig.mk	2008-01-04 21:27:16.431626182 +0100
@@ -44,7 +44,7 @@
 MV = mv -f
 
 # Fortran compiler (not normally required)
-F77 = f77
+F77 = gfortran
 F77FLAGS = -O
 F77LIB =
 
@@ -85,7 +85,7 @@
 # BLAS = -lgoto -lgfortran -lgfortranbegin
 
 # This is probably slow ... it might connect to the Standard Reference BLAS:
-BLAS = -lblas -lgfortran -lgfortranbegin
+BLAS = -lblas
 LAPACK = -llapack
 
 # The BLAS might not contain xerbla, an error-handling routine for LAPACK and
@@ -114,8 +114,8 @@
 # The path is relative to where it is used, in CHOLMOD/Lib, CHOLMOD/MATLAB, etc.
 # You may wish to use an absolute path.  METIS is optional.  Compile
 # CHOLMOD with -DNPARTITION if you do not wish to use METIS.
-METIS_PATH = ../../metis-4.0
-METIS = ../../metis-4.0/libmetis.a
+METIS_PATH = /usr/include/metis
+METIS = -lmetis
 
 # If you use CHOLMOD_CONFIG = -DNPARTITION then you must use the following
 # options:
